python/graphframes/graphframe.py

```diff
@@ -16,7 +16,7 @@
 #
 
 from pyspark import SparkContext
-from pyspark.sql import Column, DataFrame, functions as sqlfunctions, SQLContext
+from pyspark.sql import Column, DataFrame, functions as sqlFuncs, SQLContext
```

Comment:
I think having sqlfunctions and having calls as `sqlfunctions.col` `sqlfunctions.sum` make sense